### PR TITLE
Example: Intel indicators

### PR DESCRIPTION
**Pull request by isimluk**

### CODE REVIEW

File: examples/falcon_intel_indicators/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+```
+go get github.com/crowdstrike/gofalcon/examples/falcon_intel_indicators
+```
+
+## Exemplary Usage
+
+Interactive Run:
+```
+$ FALCON_CLIENT_ID="abc" FALCON_CLIENT_SECRET="XYZ" FALCON_CLOUD=us-1 \
+      falcon_intel_indicators --filter="type:'url'+malicious_confidence:'Medium'"  | jq length
```
Comment (redhatrises) [inline examples/falcon_intel_indicators/README.md:18]: `type:'url'+malicious_confidence:'Medium'` this seems special.... don't suppose there is a way to make this easier or cleaner?

Comment (isimluk) [reply examples/falcon_intel_indicators/README.md:18]: It is example just an example of a filter. Will improve the documentation to make that clear.

**Comment by isimluk** [comment]
🙏

File: examples/falcon_intel_indicators/README.md (hunk 1 above)
Comment (redhatrises) [reply examples/falcon_intel_indicators/README.md]: Thanks... was just wondering if there is a better coding solution to create a better customer experience... but that's probably something to think about for another time.

**Pull request merged**